Automattic/woocommerce-payments
# Display credit card brand icons on order received page

Fixes #8897 

#### Changes proposed in this Pull Request

This adds credit card brand icons and last 4 card digits to the order received page in the format, [Card icon] ••• 4242. 









#### Testing instructions

1. Place an order using a Visa card 
2. On the Order Received page, observe the Visa icon and last 4 card digits under the Payment Method heading
3. Place an order using another card, Mastercard `5555555555554444`, for example
4. On the Order Received page, observe the Mastercard icon and last 4 card digits under the Payment Method heading





-------------------


**Post merge**
